bytesize.go

```diff
@@ -0,0 +1,135 @@
+package userconfig
```

Comment:
Isn't there already some library for this stuff? IMO it should be a separated public package, shouldn't it?